common_test.go

```diff
@@ -250,14 +254,17 @@ func TestMain(m *testing.M) {
 		root: "./test/index",
 	}
 
+	upstreamBuf := make([]byte, base64.StdEncoding.EncodedLen(25))
```

Comment:
This shouldnt be hard-code, it may be best to define the user/pass in a variable above and get the len. That way in the future changing the test user/pass won't break this 25